bin/describe/describe_pp.ml

```diff
@@ -66,31 +65,32 @@ let get_pped_file super_context file =
         Dune_rules.Dune_load.Dune_files.in_dir (dir |> in_build_dir)
       in
       let staged_pps =
-        Option.bind dune_file ~f:(fun dune_file ->
-            dune_file.stanzas
-            |> List.fold_left ~init:None ~f:(fun acc stanza ->
-                   match stanza with
-                   | Dune_rules.Dune_file.Library lib -> (
-                     let preprocess =
-                       Dune_rules.Preprocess.Per_module.(
-                         lib.buildable.preprocess |> single_preprocess)
-                     in
-                     match preprocess with
-                     | Dune_rules.Preprocess.Pps ({ staged = true; _ } as pps)
-                       -> Some pps
-                     | _ -> acc)
-                   | _ -> acc))
+        let open Option.O in
```

Comment:
I don't think this indentation removal gained us anything. It's not even intended very much and the lines have the same wrapping.

Also, don't mix refactoring with real changes. It's hard to review this way.

Reply:
yeah fair enough, I'll revert this